tailwind.config.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * The Tailwind config uses the pre-configured config within
+ * demosplan-ui, which is tweaked with demosplan-ui design tokens.
+ * Tailwind is included as a PostCss plugin in the webpack config.
+ * The Tailwind entry point is ./client/css/index.css.
+ */
+const config = {
+  ...require('@demos-europe/demosplan-ui/tailwind.config'),
+  content: [
+    './client/js/**/!(generated|legacy|store)/*.{js,vue}',
+    './demosplan/plugins/**/Resources/**/*.twig',
+    './node_modules/@demos-europe/demosplan-ui/dist/*.js',
+    './projects/**/templates/**/*.twig',
+    './templates/bundles/DemosPlanCoreBundle/**/*.twig'
+  ],
+  safelist: [] // Can be removed when demosplan-ui drops the /./ pattern in the Tailwind config
```

Comment:
When we now generate the tailwind styles here in the core, do we still need the /./ Pattern indemosplan-ui? Or can it be removed in a next step?

Reply:
It can be removed, this is correct.